### DISCUSSION TITLE
CPU percentage shows incorrect

### DISCUSSION THREAD

**Discussion by NoisyBoy1308** [mathoudebine/turing-smart-screen-python#143] [General]
As mentioned in the title, the percentage of my 5900x CPU shows different values than win11 task manager does.
It shows 0-2% in idle where taskmanager shows 5-9%
When I stress the CPU via CPUZ it shows 89% where it should be 100%.
i guess it reads the wrong sensor. Maybe wrong core. How can I get it to show same values as task manager does?

**Comment by mathoudebine** [comment]
Hi, sorry to hear about that issue.
Are you using the latest version of this project?
Can you install and run "[LibreHardwareMonitor 0.9.1](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor)" and see if there is also a difference between the CPU% displayed by LibreHardwareMonitor and the one from Task Manager?

**Comment by ovni76** [comment]
Same Here Windows 11 has a bug for the report of cpu use.

Librehardware implement a patch.

So for Windows 11 user just replace the 2 DLL in  \external\LibreHardwareMonitor by the last one comming from the librehardware distribution.

You will have the good cpu use.

**Reply by mathoudebine** [reply]
Thanks for the info! I will update the LHM library to 0.9.2
